assets/src/components/nav/topNav.tsx

```diff
@@ -48,9 +49,23 @@ const TopNav = (): JSX.Element => {
               show={showUserPopover}
               placement="bottom"
             >
-              <Popover className="c-top-nav__popover">
-                <Popover.Body>
-                  <LoggedInAs email={email} />
+              <Popover className="c-top-nav__popover inherit-box border-box">
+                <Popover.Body className="p-0">
+                  <Dropdown.Menu
+                    show
+                    className="position-static border border-0"
+                  >
+                    <Dropdown.ItemText className="w-auto">
+                      <LoggedInAs email={email} />
+                    </Dropdown.ItemText>
+                    <Dropdown.Divider />
+                    <Dropdown.Item
+                      href="/auth/keycloak/logout"
+                      className="w-100 icon-link"
```

Comment:
(it seems like this is still in flux, I'm just noting this so it doesn't get lost)
nit(non-blocking): I'm not seeing `w-100` do anything when I remove it? 

side-note: I learned recently that `width: auto` is the default behaviour of `width` (which `auto` essentially turns into `100%`), and often `width: 100%` is redundant, so I'm wondering if that applies here.

Reply:
Ah, I think that and the `w-auto` are vestigial then, I'll remove them.

Reply:
9ddc5815